src/sentry/static/sentry/app/views/organizationActivity/activityFeedItem.tsx

```diff
@@ -301,104 +299,127 @@ class ActivityItem extends React.Component {
   };
 
   render() {
-    const item = this.props.item;
-
-    let bubbleClassName = 'activity-item-bubble';
-    if (this.state.clipped) {
-      bubbleClassName += ' clipped';
-    }
+    const {className, item} = this.props;
 
     const avatar = item.user ? (
-      <UserAvatar user={item.user} size={36} className="activity-avatar" />
+      <UserAvatar user={item.user} size={36} />
     ) : (
-      <div className="activity-avatar avatar sentry">
-        <span className="icon-sentry-logo" />
-      </div>
+      <IconSentry size="36px" />
```

Comment:
Should this be a rounded icon like we use on discover query cards and issue activity?

Reply:
Updated to re-use `<ActivityAvatar>`